ingestion/src/metadata/ingestion/source/dashboard/tableau/queries.py

```diff
@@ -16,58 +16,66 @@
 TABLEAU_SHEET_QUERY_BY_ID = """
 query SheetQuery {{
   sheets(filter: {{luid: "{id}" }} ) {{
+    ...SheetFields
+  }},
+  dashboards(filter: {{luid: "{id}" }} ) {{
+    sheets {{
+      ...SheetFields
+    }}
+  }}
+}}
+fragment SheetFields on Sheet {{
```

Comment:
This fragment was added to avoid repeating the sheet fields on the 'base' sheets and dashboard sheets.